lola.html

```diff
@@ -635,29 +639,45 @@ <h3>Authorization</h3>
         suspension. 
       </p>
 
-
       <h3>Migration Outbox</h3>
+	  <p>
+		The migration outbox is intended as a simple way for applications to enable LOLA account migration.  Servers implementing LOLA MUST include this collection in actor's profiles, as it will provide maximum compatibility with other ActivityPub software.
+		This collection SHOULD contain a filtered view of all the actor's activities that the source server deems eligible for migration.  The source server MAY filter out activities that are not relevant to the destination server, or that the user has chosen to exclude during the OAuth authorization step.
```

Comment:
Remember that HTML source formatting doesn't get reflected in the HTML rendering. If it's worth breaking paragraphs in the source, it's probably worth breaking them for the rendering.
```suggestion
		The migration outbox is intended as a simple way for applications to enable LOLA account migration.  Servers implementing LOLA MUST include this collection in actor profiles, as it will provide maximum compatibility with other ActivityPub software.
	  </p>
      <p>
        This collection SHOULD contain a filtered view of all the actor's activities that the source server deems eligible for migration.  The source server MAY filter out activities that are not relevant to the destination server, or that the user has chosen to exclude during the OAuth authorization step.
```
(It likely makes sense to keep all lines of the HTML source ~80 chars or less, but that's a separate consideration.)

Reply:
Sure, this makes sense.  I usually just view the document with "word wrap", but I can follow whatever standard is in place here.

Speaking of HTML code formatting, there's a lot of small inconsistencies throughout the document.  Further down the road, I was planning to ask if some sort of automated code formatter might be useful. Since we're here, I'm mentioning it now - would that be useful to this project?